app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/ContractDetailDestination.kt

```diff
@@ -158,28 +163,35 @@ private fun ContractDetailScreen(
                 when (pageIndex) {
                   0 -> {
                     YourInfoTab(
-                      coverageItems = state.contractDetails.overviewItems,
-                      allowChangeAddress = state.contractDetails.allowChangeAddress,
-                      allowEditCoInsured = state.contractDetails.allowEditCoInsured,
+                      coverageItems = state.insuranceContract.currentAgreement.displayItems
+                        .map { it.title to it.value }
+                        .toImmutableList(),
+                      allowEditCoInsured = true, // TODO
+                      allowChangeAddress = true, // TODO
```

Comment:
Are these two just not present in the new Api then somehow?
We had this https://github.com/HedvigInsurance/android/blob/23cd4e0bb5e7c6678c5fa4c54631a01b5a230616/app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/data/GetContractDetailsUseCase.kt#L162-L207 for the co-insured part before, can we have it again here? Do we not want to have it anymore perhaps? Is that the case?

Reply:
Good catch. Im changing so that we look at ContractType for co insured. For address change we actually get from the backend now.